R-packages/evalcast/R/get_covidhub_predictions.R

```diff
@@ -384,7 +384,11 @@ get_forecaster_predictions_alt <- function(covidhub_forecaster_name,
       collect() %>%
       left_join(target_separated, by = "target") %>%
       select(-.data$target) %>%
-      mutate(forecaster = covidhub_forecaster_name) %>%
+      mutate(forecaster = covidhub_forecaster_name,
+             forecast_date = as.Date(forecast_date, format = "%Y-%m-%d"),
+             target_end_date = as.Date(target_end_date, format = "%Y-%m-%d"),
```

Comment:
Can you use `lubridate::ymd()` for these? Seems a bit more robust. (I think `ymd()` is already imported. 

Also, when you mutate, it can change the ordering of the columns. Can you ensure that they remain in the same order?